config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  config.default_per_page = 10
```

Comment:
can we add a quick test where we stub this number to be something like 1 and make sure that a second record doesn't show? Not a huge deal, just would be nice to have a smoke test that pagination is working